fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/controller/ViolationsController.java

```diff
@@ -182,6 +187,14 @@ public Violation resolveViolations(
         return entityToDto.convert(dbViolationEntity);
     }
 
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Violation created successfully!")})
```

Comment:
actually, we don't know if the violation will be created. It might be a duplicate or it might be whitelisted. The processing of the ViolationSink happens asynchronously.
Probably it makes sense to use the [202 http status](https://httpstatuses.com/202) and just return en empty response.